.github/workflows/dotnetcore.yml

```diff
@@ -41,14 +44,24 @@ jobs:
           path: '**/*.trx'
           reporter: dotnet-trx
 
-    # PR opened, push prelease Carter package to feedz
+    - if: >-
+        (github.event_name == 'pull_request')
```

Comment:
Don't think we'd want to pack the template on a pull request?

Reply:
No?
My thought was that having both the packages pushed at the same time would allow to test them.

Your call, happy to remove and only push a stable package when a tag is created on `main`.

Reply:
No? My thinking was that always having both packages published at the same time would allow testing them both.

It's your call, really, pushing the template package only when a tag is created on `main` is doable.

Reply:
With #305, we stopped publishing packages when a PR is open, only when a PR is merged or the repo is tagged.